frontend/src/main/components/Jobs/PagedJobsTable.js

```diff
@@ -30,6 +30,8 @@ export default function PagedJobsTable() {
     );
     // Stryker restore  all
 
+    page.content = jobsProp || page.content;
```

Comment:
Don’t mutate this.

Reply:
Yeah, it's weird to mutate the variable that's being returned from a React Query call, especially if the purpose is for storybook.

We have a pattern for that, and that pattern is using msw (see the examples in the team03 assignment.)